src/main/java/com/crud/tasks/TasksApplication.java

```diff
@@ -4,9 +4,15 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
 @SpringBootApplication
+//public class TasksApplication extends SpringBootServletInitializer {
```

Comment:
co to za komentarze? nie wrzucamy do repo komentarzy bez dodatkowego uzasadniajacego komentarza